test/unit/acl/access-groups.test.ts

```diff
@@ -0,0 +1,12 @@
+jest.mock('rdflib')
+import * as RdfLib from 'rdflib'
+jest.mock('solid-auth-client')
+import * as SolidAuthClient from 'solid-auth-client'
```

Comment:
These imports are unused? (Likewise in other tests.)

Reply:
You need to import them for jest.mock to work, right?

Reply:
No, `jest.mock` makes sure that whenever the code under test imports the given module, that it will receive the mocked replacement. If you import then yourself, I think that just means you have an instance of the mocks in your test file, but you're not using them anywhere.

Reply:
Ah ok! Good to know, I'll remove them tomorrow.

Reply:
Done